app/controllers/confirmations_controller.rb

```diff
@@ -11,18 +11,34 @@ def show
   end
 
   def confirm
+    confirmation_token = params[resource_name][:confirmation_token]
+    @resource = resource
     digested_token = Devise.token_generator
-                         .digest(self, :confirmation_token, params[resource_name][:confirmation_token])
+                         .digest(self, :confirmation_token, confirmation_token)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.